README.md

```diff
@@ -41,23 +48,38 @@ resolved Promise.
 
 ### Generate a DID Document
 
-Generate a new DID document.
+#### Generating and Registering a Veres One DID Document
 
-* options - a set of options used when generating the DID Document
-  * didType - the type of DID to generate.
-      Options: 'nym' or 'uuid' (default: 'nym')
-  * keyType - the type of keys to generate.
-      Options: 'RsaVerificationKey2018' (default: 'RsaVerificationKey2018').
-  * passphrase - the passphrase to use to encrypt the private keys for
-      nym-based DIDs. Set to `null` if the private keys should not be encrypted.
-  * mode - the mode/environment to generate the DID in.
-      Options: 'dev', 'test', 'live' (default: 'dev').
+```js
+// Generate a new DID Document
+const didDocument = await veresDriver.generate(
+  {didType: 'nym', keyType: 'Ed25519VerificationKey2018'}); // default
+
+// Log the new didDocument to the console.
+console.log('Generated:', JSON.stringify(didDocument, null, 2));
+
+// Now register the newly generated DID Document
+const registrationResult = await veresDriver.register({didDocument});
+
+// Log the result of registering the didDoc to the VeresOne Test ledger
+console.log('Registered!', JSON.stringify(registrationResult, null, 2));
+```
 
-Returns an object with:
+#### Registering a (newly generated) DID Document
 
-* publicDidDocument - the generated DID Document.
-* privateDidDocument - the DID Document augmented with the encrypted private
-    keys in PEM format.
+To register a DID Document:
+
+```js
+const registrationResult = await veresDriver.register({didDocument});
+```
+
+#### `get()` - Retrieving a Veres One DID Document
```

Comment:
I don't think the "get()" part needs to be in the section title.  Other sections don't do that.

Reply:
Thanks.